hazelcast-jet-core/src/main/java/com/hazelcast/jet/Processors.java

```diff
@@ -843,6 +843,15 @@ public static ProcessorMetaSupplier writeFile(@Nonnull String directoryName) {
     }
 
     /**
+     * Returns a supplier of processor that do nothing. They consume all input
```

Comment:
Pushed a fix. Hope I won't cause too many conflicts with your changes. I only changed javadoc of sources/sinks, that you are not going to change.